### PR TITLE
Feat/simple community handles

### PR DESCRIPTION
**Pull request by friedger**
This PR
* simplifies the community handle contract by removing the ownership
* adds name-preorder to approving-namespace-controller
* add name-renewal to approving-namespace-controller (limited to 100 renewals)

### CODE REVIEW

File: backend-public/contracts/approving-namespace-controller.clar [hunk 1]
```diff
@@ -34,8 +83,11 @@
                               (zonefile-hash (optional (buff 20))))
     (let ((price (var-get price-in-ustx))
           (owner tx-sender)
-          (hash (sha256 (concat (concat (concat name 0x2e) namespace) name-salt))))
+          (last-salt (unwrap! (map-get? renewal-salts name) err-not-found))
+          (salt (unwrap! (as-max-len? (concat last-salt 0x00) u100) err-max-renewal-reached))
```
Comment (MarvinJanssen) [inline backend-public/contracts/approving-namespace-controller.clar]: Why max renewals?

File: backend-public/contracts/community-handles.clar [hunk 2]
```diff
@@ -37,22 +38,31 @@
                                 namespace-ready namespace)))
         (ok true)))
 
+(define-public (namespace-revoke-function-price-edition (namespace (buff 20)))
```
Comment (MarvinJanssen) [inline backend-public/contracts/community-handles.clar]: Good idea to add.

File: backend-public/contracts/community-handles.clar [hunk 3]
```diff
@@ -37,22 +38,31 @@
...
+    (begin
+        (try! (is-contract-caller-namespace-controller namespace))
+        (try! (ft-mint? danger-zone-token u1 tx-sender))
+        (try! (ft-burn? danger-zone-token u1 tx-sender))
```
Comment (MarvinJanssen) [inline backend-public/contracts/community-handles.clar]: I love this. SIP-worthy.

**Review by MarvinJanssen — APPROVED** [review]
Looks good! I like the danger token.

(Reviewed on mobile device)

File: backend-public/contracts/approving-namespace-controller.clar (hunk 1 above)
Comment (friedger) [reply backend-public/contracts/approving-namespace-controller.clar]: I have removed that now.

File: backend-public/contracts/community-handles.clar (hunk 2 above)
Comment (friedger) [reply backend-public/contracts/community-handles.clar]: :+1:

**Pull request merged**